src/formio/components/Children/Children.scss

```
@@ -0,0 +1,22 @@
+.openforms-children {
+  padding: 1em;
+
+  @media (max-width: 768px) {
```

```suggestion
  @media (max-width: $breakpoint-tablet) {
```
let's use our variables for the breakpoints instead of magic numbers so that the viewport handling is consistent